src/lib.h

```diff
@@ -99,7 +99,7 @@ typedef struct C_VerificationSignature C_VerificationSignature;
  * If something goes wrong, this will return a null pointer. Don't forget to
  * destroy the returned pointer once you are done with it!
  */
-C_BatchDLEQProof *batch_dleq_proof_decode_base64(const char *s);
+C_BatchDLEQProof *batch_dleq_proof_decode_base64(const uint8_t *s, uintptr_t s_length);
```

Comment:
Passing pointers and lengths was also how we handled FFI between Rust and C in tor.